brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ApplicationSpecsBuilder.java

```diff
@@ -94,6 +94,7 @@ public ApplicationSpecsBuilder(EntitySpecFactory entitySpecFactory, Collection<E
         visited.add(node);
 
         EntitySpec<?> spec = createSpec(node, nodeTemplate, topology);
+        LOG.info("applying spec modifiers {}", specModifiers);
```

Comment:
This is debug at best, probably trace, and should include the spec/nodeTemplate in question. One to tweak subsequently.